cfme/tests/infrastructure/test_config_management_rest.py

```diff
@@ -230,3 +231,66 @@ def test_authentications_options(self, appliance, config_manager):
         collection = appliance.rest_api.collections.authentications
         assert 'credential_types' in collection.options()['data']
         assert_response(appliance)
+
+
+@pytest.fixture(scope="module")
+def config_manager_rest(config_manager_obj):
+    """Creates provider using REST API."""
+    config_manager_obj.create_rest()
+    assert_response(config_manager_obj.appliance)
+    rest_entity = config_manager_obj.rest_api_entity
+    rest_entity.reload()
+
+    yield rest_entity
+
+    if rest_entity.exists:
+        rest_entity.action.delete()
+        rest_entity.wait_not_exists()
+
+
+@pytest.mark.tier(3)
+def test_config_manager_create(config_manager_rest):
```

Comment:
Please add your rest requirement.

Reply:
We have a requirement `tower` in pytestmark, do we need to add rest requirement there? And if yes, we'll have to add it to the whole file.

Reply:
Its depends on you. Decided this test you want to put under `tower` FA or `rest`. Global mark will mark it as `tower` else add local marker.